src/logging/Manager.cc

```diff
@@ -743,6 +744,15 @@ bool Manager::Write(EnumVal* id, RecordVal* columns_arg)
 				continue;
 			}
 
+		if ( global_log_policy_hook )
```

Comment:
I considered iterating over the two hooks (`filter->policy` and `global_log_policy_hook`) via for-loop, but then you lose the convenient `continue` to skip the entry. So I think that bit of redundancy is okay here.